benchmark.js

```diff
@@ -10,29 +20,28 @@ const parse = compile(schema, {optional: false, validate: false, unsafe: false,
 
 const parseNoBuf = compile(schema, {optional: false, validate: false, unsafe: false, unescapeStrings: false})
 
-
 console.log('Generated code:')
 console.log(parse.toString())
 console.log('One parse:')
 console.log(parse(s, b))
 
 const cnt = 3e7
+var i
```

Comment:
If `const`, why `var` not `let`?

Reply:
This change is made because the `i` variable is defined multiple times in this file. I just copy-paste the variable definition

Reply:
With `let` it'll be scoped to the `for` loop if you put the definition into the loop initializer where it was, so there won't be a naming collision. Are you concerned with the transpiled output of the library having two variables?